docker/run.sh

```diff
@@ -9,9 +9,9 @@ if [ -n "$(ls /host/lib/libsystemd* 2>/dev/null)" ]; then
   ln -s /host/lib/libsystemd* /lib/x86_64-linux-gnu/
 fi
 
-if [ -n "$METADATA_AGENT_HOSTNAME" ]; then
```

Comment:
For backward compatibility reasons, let's make it something like this:
```sh
if [ -z "${METADATA_AGENT_URL:-}" -a -n "${METADATA_AGENT_HOSTNAME:-}" ]; then
  METADATA_AGENT_URL="http://${METADATA_AGENT_HOSTNAME}:8000"
fi
if [ -n "${METADATA_AGENT_URL}" ]; then
  sed -i ...
fi
```

Reply:
Done